docs/source/testing-and-debugging.md

```diff
@@ -62,24 +69,24 @@ from boa3_test.test_drive.testrunner.neo_test_runner import NeoTestRunner
 
 
 def test_hello_world_main():
-    neoxp_folder = '{path-to-neo-express-directory}'
+    neoxp_config_file = '{path-to-neo-express-config-file}'
     project_root_folder = '{path-to-project-root-folder}'
     path = f'{project_root_folder}/boa3_test/examples/hello_world.nef'
-    runner = NeoTestRunner(neoxp_folder)
+    runner = NeoTestRunner(neoxp_config_file)
 
     invoke = runner.call_contract(path, 'main')
     runner.execute()
```

Comment:
This apparently can return an error message. If it does, then inspecting `invoke.result` is going to be really confusing. For example, I was calling a wrong function (=wrong name, didn't exist). What `invoke.result` shows is
![image](https://github.com/CityOfZion/neo3-boa/assets/6625537/0b061930-6cc5-43d3-8642-34e79b28fa57)

I wanted to add the comment that we should promote error check prior to asserting the results. However, that is going to be somewhat hard because the return type is something like `Union[str, list[NeoInvokeResult]]` where `str` is an exception message. So that means one has to do something like
```python
result = runner.execute()
if isinstance(result, str):
   raise Exception(result)
assert invoke.result is None
```

What I personally think would have been prettier is
```python
success, err_msg = runner.execute()
if not success:
   raise Exception(err_msg)
assert invoke.result is None
```

Alternatively (and maybe even better) is that `runner.execute()` throws an exception if it fails. Under the assumption that we eventually move to a unittest like class you could then do scenarios like
```python
with self.assertRaises(ValueError) as context:
   runner.execute()
self.assertIn("method not found", str(context.exception))
```
and maybe even nicely handle in contract asserts (if [this PR](https://github.com/neo-project/neo-vm/pull/491) ever gets accepted)